tslint.json

```diff
@@ -83,7 +83,13 @@
     "template-banana-in-box": true,
     "template-no-negated-async": true,
     "use-lifecycle-interface": true,
-    "use-pipe-transform-interface": true
+    "use-pipe-transform-interface": true,
+    "variable-name": {
+      "options": [
+        "allow-snake-case",
+        "allow-pascal-case"
+      ]
```

Comment:
This is what is in master and that you approved in a PR.

Reply:
okay